app/src/main/java/org/simple/clinic/summary/teleconsultation/api/TeleconsultInfo.kt

```diff
@@ -9,7 +9,10 @@ sealed class TeleconsultInfo : Parcelable {
   object Fetching : TeleconsultInfo()
 
   @Parcelize
-  data class Fetched(val doctorPhoneNumber: String) : TeleconsultInfo()
+  data class Fetched(
+      val doctorPhoneNumber: String,
+      val phoneNumbers: List<TeleconsultPhoneNumber>
```

Comment:
The name `phoneNumbers` when contrasted with `doctorPhoneNumber` indicates that it is not a `doctorPhoneNumber` which is not true. We should rename the properties to better expose this, say `primaryPhoneNumber` and `alternatePhoneNumbers` or something like that.

Reply:
Yeah, eventually we will remove the `doctorPhoneNumber` variable since the primary phone number will be part of the phone numbers list.

Reply:
Works.